daphne/src/testing.rs

```diff
@@ -1136,6 +1204,8 @@ impl AggregationJobTest {
             prometheus_registry,
             leader_metrics,
             helper_metrics,
+            leader_reports_processed: Arc::new(Mutex::new(HashSet::new())),
+            helper_reports_processed: Arc::new(Mutex::new(HashSet::new())),
```

Comment:
just a pro tip: you can write `Default::default()` here instead of all this, and if you change the types later it will probably still work and you'll have to change less code